pyccel/codegen/wrapper/fortran_to_c_wrapper.py

```diff
@@ -259,8 +205,10 @@ def _wrap_FunctionDef(self, expr):
             return EmptyNode()
 
         # Wrap the arguments and collect the expressions passed as the call argument.
-        func_arguments = [self._wrap(a) for a in expr.arguments]
-        call_arguments = [self._get_call_argument(fa) for fa in func_arguments]
+        wrapped_args = [self._extract_FunctionDefArgument(a, expr) for a in expr.arguments]
+        #func_arguments = [self._wrap(a) for a in expr.arguments]
```

Comment:
Do you want to keep this old line as a comment? I have nothing against it if you think it is useful

Reply:
No it has just been forgotten
```suggestion
```